API/docs/validation.md

```diff
@@ -0,0 +1,60 @@
+# Panalpina Bugfix#1 verfiication
+
+The fix is only for backend the verification document explains only aboout how to verify the backend service.
+
+
+#Apply patch
+
+Skip this test if already done.
+
+git am > 0001-commit-for-BugFix-1-challenge.patch
```

Comment:
Your typo will truncate the patch file to a zero-length file.

==> `git am < 0001-commit-for-BugFix-1-challenge.patch`

Reply:
but am able to apply the patch successfully with warnings only. Can you please help me to understand?
![image](https://user-images.githubusercontent.com/2901614/53809065-de056900-3f79-11e9-8488-8111b1026ddd.png)

Reply:
git am > ... is wrong.

> != <

Reply:
Okay.. let me read about it..

Reply:
There is nothing to read about ...

`git am > 0001-commit-for-BugFix-1-challenge.patch`

The command above is DIFFERENT from the one in your screen shot because you used `>` instead of `<`.

Reply:
Ok got it.. I raised a new PR for my changes as per challenge spec.. Can you please look into it?